### ISSUE TITLE
How can we user sign_up or login and so on without authenticate,

### ISSUE THREAD

**Issue by fkoyo** [graphql-devise/graphql_devise#157]
I want to user mutation before authenticating for example sign_up, login,  confirm_account, and so on.
what do I have to do?

I post my setting following:

graphql/my_schema.rb
```
class BusinessCloudSchema < GraphQL::Schema
  use GraphqlDevise::SchemaPlugin.new(
    query:            Types::QueryType,
    mutation:         Types::MutationType,
    resource_loaders: [
      GraphqlDevise::ResourceLoader.new(
        'User',
        only: [
          :sign_up,
          :login,
          :confirm_account,
          :send_password_reset
        ]
      ),
    ],
  )
  mutation(Types::MutationType)
  query(Types::QueryType)

  # Setup GraphQL to use the built-in lazy_resolve method
  use BatchLoader::GraphQL
end
```

mutation_type.rb
```
module Types
  class MutationType < Types::BaseObject
    field :register_organization,   mutation: Mutations::RegisterOrganization
    field :update_user,             mutation: Mutations::UpdateUser
  end
end
```

**Comment by fkoyo** [comment]
I get it. If we add the following my_schema.
in  ```/graphiql``` URL, the docks present the no schema, but this is the right way to define pre login action.

```
class BusinessCloudSchema < GraphQL::Schema
  use GraphqlDevise::SchemaPlugin.new(
    query:            Types::QueryType,
    mutation:         Types::MutationType,
    resource_loaders: [
      GraphqlDevise::ResourceLoader.new(
        'User',
        only: [
          :sign_up,
          :login,
          :confirm_account,
          :send_password_reset
        ]
      ),
    ],
  )
  mutation(Types::MutationType)
  query(Types::QueryType)

  # Setup GraphQL to use the built-in lazy_resolve method
  use BatchLoader::GraphQL
end

```

![image](https://user-images.githubusercontent.com/18605223/109845075-18e92700-7c90-11eb-9b06-b3a538e721a3.png)

**Comment by mcelicalderon** [comment]
@fkoyo not sure if you saw this, but #154 allows unauthenticated introspection queries. By default, introspection is public in all envs but production by default (`!Rails.env.production`). We still need to document this, but let me know if that makes sense or if something is unclear. The PR simply adds a `public_introspection` param to the plugin initializer where you can specify this. This is available in `v0.14.2`.
